servers/publikator/lib/mailchimp/updateCampaignContent.js

```diff
@@ -10,12 +10,16 @@ module.exports = async ({ campaignId, html }) => {
     method: 'PUT',
     headers: {
       'Content-Type': 'application/json',
-      'Authorization': `Basic ${Buffer.from('anystring:' + MAILCHIMP_API_KEY).toString('base64')}`
+      Authorization: `Basic ${Buffer.from('anystring:' + MAILCHIMP_API_KEY).toString('base64')}`
     },
     body: JSON.stringify({
       html
     })
   })
-    .then(response => response.json())
-    .catch(error => console.error('updateMailchimp failed', { error }))
+    .then(response => {
+      if (!response.ok) {
+        throw Error(response.statusText)
+      }
+      return response
+    })
```

Comment:
Before, the response was returned as json, and I think that's nice of a lib to do instead of returning a fetch-respose. Also there was [a check](https://github.com/orbiting/backends/pull/436/files#diff-6e358742c5bc46972ccb65ea7e02ad7fL457) for `response.status` to be missing before. Maybe it would be good to keep this check.

Reply:
`response.status` was good, `response.ok` fits even better in case of MailChimp API.

Their API really sticks to (resource) status codes. Whatever you `POST`, `PATCH`, `PUT` you will either result in a good status code (200-299, thus `response.ok` becomes truthy) or bad one.